src/components/templates/PosterPage/components/ShareModal/ShareModal.scss

```diff
@@ -0,0 +1,82 @@
+@import "scss/constants.scss";
+
+.ShareModal {
+  &__header {
+    position: relative;
+    height: 200px;
+    margin: $spacing--sm;
+    border-radius: $border-radius--lg;
+  }
+
+  &__background {
+    z-index: z(map-background);
```

Comment:
Why is the `map-background` `z-index` layer being used here? That seems wrong..? If this needs a `z-index` layer at all, then it should probably be using it's own?

Reply:
@oleksii-front This still seems to be outstanding/requires answering?

Reply:
Was removed